test/unit/ConfigurationFactoryTest.php

```diff
@@ -4,24 +4,32 @@
 
 use Assertis\Configuration\Collection\ConfigurationArray;
 use Assertis\Configuration\Drivers\SourceDriver;
-use Silex\Application;
-use Silex\Provider\ValidatorServiceProvider;
 use Symfony\Component\Validator\Constraints as Assert;
+use Symfony\Component\Validator\ConstraintValidatorFactory;
+use Symfony\Component\Validator\Mapping\Factory\LazyLoadingMetadataFactory;
+use Symfony\Component\Validator\Mapping\Loader\StaticMethodLoader;
+use Symfony\Component\Validator\Validation;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
 
 /**
  * @author Maciej Romanski <maciej.romanski@assertis.co.uk>
  */
 class ConfigurationFactoryTest extends \PHPUnit_Framework_TestCase
 {
     /**
-     * @var Application
+     * @return ValidatorInterface
      */
-    private $app;
-
-    public function setUp()
+    private function getValidator()
```

Comment:
Do you think we will need method like this in more places? If yes, maybe we should put it into Util repo or smth?